src/app/lib/layoutGrid/index.js

```diff
@@ -168,7 +169,7 @@ export const layoutGridItemSmall = css`
     grid-column: 1 / span 5;
     max-width: ${fiveOfSixColumnsMaxWidthScaleable};
     padding: 0
-      ${props => (props.padding && props.padding.group3) || 'auto'};
+      ${props => deepGet(['padding', 'group3'], props) || ''};
```

Comment:
```suggestion
      ${({ padding }) => padding.group3 || ''};
```

Reply:
We would still need to check padding is not undefined though.

We can still use de-structuring but I don't see any benefit because it would make those lines of code longer.

Reply:
Ah yes, apologies, it'd need to be ` ${({ padding }) => (padding && padding.group3) || ''};`.  Having discussed [an upcoming issue this morning](https://github.com/bbc/simorgh/issues/2149#issuecomment-507219978) that will shortly remove these bespoke props entirely, I'm happy to have this merged in as it currently is.